terraform/alerting/notifications.tf

```diff
@@ -1,12 +1,25 @@
-resource "google_monitoring_notification_channel" "channels" {
+resource "google_monitoring_notification_channel" "paging" {
   provider     = google-beta
   project      = var.project
-  display_name = "${each.key} Notification Channel"
+  display_name = "Paging Notification Channel"
   type         = each.key
   labels       = each.value.labels
   depends_on = [
     null_resource.manual-step-to-enable-workspace
   ]
 
-  for_each = var.alert-notification-channels
+  for_each = var.alert-notification-channel-paging
+}
+
+resource "google_monitoring_notification_channel" "nonpaging" {
```

Comment:
Nit: It's standard to use dashes for separators in resource names (e.g. this should be `"non-paging"`)

Reply:
Addressed, thank you.